cobalt/uri.py

```diff
@@ -131,11 +137,16 @@ def uri(self):
 
     def work_uri(self, work_component=True):
         """ String form of the work URI. """
+        prefix = self.prefix
```

Comment:
no real benefit to assigning this to another variable, just use `self.prefix` later one